tools/cldr-code/src/main/java/org/unicode/cldr/util/PathHeader.java

```diff
@@ -2116,11 +2107,24 @@ public static String getMetazonePageTerritory(String source) {
 
     private static final List<String> COUNTS = Arrays.asList("displayName", "zero", "one", "two", "few", "many", "other", "per");
 
+    private static Collator alphabetic;
+
     private static int alphabeticCompare(String aa, String bb) {
-        // A frozen Collator is thread-safe.
+        if (alphabetic == null) {
+            initializeAlphabetic();
+         }
         return alphabetic.compare(aa, bb);
     }
 
+    private static synchronized void initializeAlphabetic() {
+        // Lazy initialization: don't call CLDRConfig.getInstance() too early or we'll get
+        // "CLDRConfig.getInstance() was called prior to SurveyTool setup" when called from
+        // com.ibm.ws.microprofile.openapi.impl.core.jackson.ModelResolver._addEnumProps
+        if (alphabetic == null) {
+            alphabetic = CLDRConfig.getInstance().getCollatorRoot();
+        }
+    }
+
```

Comment:
make it a [Bill Pugh singleton](https://en.wikipedia.org/wiki/Singleton_pattern#Lazy_initialization) instead?

Reply:
you mean like
```
public final class SingletonBillPugh {
	private static class SingletonHelper {
		private static final SingletonBillPugh INSTANCE = new SingletonBillPugh();
	}

	private SingletonBillPugh() {
	}

	public static SingletonBillPugh getInstance() {
		return SingletonHelper.INSTANCE;
	}
}
```
?

Reply:
Let's discuss -- in the meantime I'll merge the PR as-is -- thanks for the review!